### PR TITLE
Add documentation for 3D Map views in layouts

### LINKED ISSUE #2900: [FEATURE] 3D maps in print layouts (part 2) - PR #7705
Original commit: https://github.com/qgis/QGIS/commit/504cd618adbef707df5f55e6d397b5dc76e6aaee by web-flow

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by havatv**
Goal: Add documentation for 3D Map views in layouts

Ticket(s): Fix #2900

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+   :align: center
+   :width: 20em
+
+   3D Map Item Properties
+
+
+.. _`layout_3dmap_scene_settings`:
+
+Scene setting
+-------------
```
Comment (DelazJ) [inline docs/user_manual/print_composer/composer_items/composer_map3d.rst:38]:
```suggestion
Scene settings
--------------
```

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+.. _`layout_3dmap_camera_pose`:
+
+Camera pose
+-----------
+
+* :guilabel:`Center X` sets the X value of the camera center
+* :guilabel:`Center Y` sets the Y value of the camera center
+* :guilabel:`Center Z` sets the Z value of the camera center
```
Comment (DelazJ) [inline docs/user_manual/print_composer/composer_items/composer_map3d.rst:56]:
```suggestion
* :guilabel:`Center X` sets the X coordinate of the camera center
* :guilabel:`Center Y` sets the Y coordinate of the camera center
* :guilabel:`Center Z` sets the Z coordinate of the camera center
```
?

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+.. _`layout_3dmap_camera_pose`:
+
+Camera pose
+-----------
+
+* :guilabel:`Center X` sets the X coordinate of the point the camera is pointing at
+* :guilabel:`Center Y` sets the Y coordinate of the point the camera is pointing at
+* :guilabel:`Center Z` sets the Z coordinate of the point the camera is pointing at
+* :guilabel:`Distance` sets the distance from the camera center to
```
Comment (havatv) [inline docs/user_manual/print_composer/composer_items/composer_map3d.rst:54]:
```suggestion
* :guilabel:`Distance` sets the distance from the camera center to the
  point the camera is pointing at
```

**Comment by havatv** [comment]
> I was not able to see what triggers the change of the print_composer_blank screenshot but if you plan to backport to 3.10, there are a couple of tools that are not suitable. Also, maybe only in my head, it was meant to show the layout without any single interaction (no options collapsed) but that's not a big deal.

I will keep that in mind when backporting.
Regarding the collapsed / not collapsed settings groups, I thought it was OK to show them collapsed, to give the reader an overview of the groups. I have now updated the screenshot to what I get when I do `New Print Layout...`.

**Comment by havatv** [comment]
> I was not able to see what triggers the change of the print_composer_blank screenshot

Wanted one with the *Add 3D Map* button included.

**Comment by havatv** [comment]
The print_composer_blank.png screenshot should now be OK for both master and 3.10.

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+Camera pose
+----------------------------------------------------------------------
+
+* :guilabel:`Center X` sets the X coordinate of the point the camera
+  is pointing at
+* :guilabel:`Center Y` sets the Y coordinate of the point the camera
+  is pointing at
+* :guilabel:`Center Z` sets the Z coordinate of the point the camera
+  is pointing at
```
[the remark this answers is not in the document]
Comment (havatv) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:57]: > Are you sure it's the coordinate of the point being viewed, and not the coordinate of the camera itself, ie where you are looking at the scene from?

I checked the .h file from the QGIS code, and "copied" what was documented there, so I hope that is correct.

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+* :guilabel:`Distance` sets the distance from the camera center to
+  the point the camera is pointing at
+* :guilabel:`Pitch` sets the rotation of the camera around the X-axis
+  (vertical rotation).
+  Values from 0 to 360 (degrees).
+  0°: terrain seen straight from above;
+  90°: horizontal (from the side);
+  180°: straight from below;
+  270°: horizontal, upside down;
+  360°: straight from above.
```
[the remark this answers is not in the document]
Comment (havatv) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:67]: > In html, these render in one line. Would look weird with all these semi-colons in a sentence (I'm not sure it's an academic use of semi-colon)

I know that they will be rendered on one line. That is why I used the semi-colon.

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:57]: Can you point the url, please? I've been looking at it before I commented and what I found was not clear enough. Thanks.

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (DelazJ) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:67]: Not sure that's a good practice.

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (havatv) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:57]: > Can you point the url, please? I've been looking at it before I commented and what I found was not clear enough. Thanks.

src/3d/qgscamerapose.h

File: docs/user_manual/print_composer/composer_items/composer_map3d.rst (hunk 5 above)
[the remark this answers is not in the document]
Comment (havatv) [reply docs/user_manual/print_composer/composer_items/composer_map3d.rst:67]: It is normal practice.

**Pull request merged**